### PR TITLE
Fix charts & data for emoji 13.1

### PR DESCRIPTION
**Pull request by macchiati**
Accumulated fixes for problems in the charts. Sorry that I didn't separate them into separate PRs.

Most of the changes are 
* updating the version to 13.1
* updates to the internal data files to fix resulting generated files
* retiring or working around some awful hacks to deal with the structure of the skintones with multiperson emoji
* a couple of XUnicodeSet.java chunks of code used in testing.
* some restructuring of code while debugging problems, with some tests for particular cases (just for debugging)
* fixing some directories, since the structure changed when moving to github
* a few updates to the header text for candidate charts
* restucturing ProposalData.SINGLETON because of order-of-static-initialization problems.

### CODE REVIEW

File: unicodetools/org/unicode/tools/emoji/Emoji.java [hunk 1]
```diff
@@ -62,6 +62,7 @@
     /**
      * Constants for versions
      */
+    public static final VersionInfo VERSION14 = VersionInfo.getInstance(13,1);
```
Comment (pedberg-icu) [inline unicodetools/org/unicode/tools/emoji/Emoji.java:65]: Was this going to get fixed? (especially since VERSION_BETA below refers to it)

**Review by pedberg-icu — COMMENTED** [review]
Emoji.java still seems to have
```
VersionInfo VERSION14 = VersionInfo.getInstance(13,1);
```
Otherwise seems OK

**Comment by macchiati** [comment]
Will fix, thanks

**Comment by macchiati** [comment]
Just fixed the version, and the proposal data.

**Review by pedberg-icu — APPROVED** [review]

**Pull request merged**